src/base_alloc/base_alloc_windows.c

```diff
@@ -13,9 +13,8 @@ void *ba_os_alloc(size_t size) {
 
 void ba_os_free(void *ptr, size_t size) { VirtualFree(ptr, 0, MEM_RELEASE); }
 
-size_t ba_os_get_page_size(void) { return 4096; /* TODO */ }
-
-//
-// TODO:
-// 1) implement ba_os_get_page_size()
-//
+size_t ba_os_get_page_size(void) {
+    SYSTEM_INFO SystemInfo;
+    GetSystemInfo(&SystemInfo);
```

Comment:
is there a reason why linux version is called under `util_init_once` and this one is not? is this a "costly" call...?